lib/features/home/home_screen.dart

```diff
@@ -1,13 +1,75 @@
 import 'package:flutter/material.dart';
+import 'package:glint_frontend/design/common/app_colours.dart';
```

Comment:
lil suggestions. i made exports for most of the things you can just package/design/exports.dart it 
:)